mlrun/__main__.py

```diff
@@ -802,15 +827,26 @@ def db(port, dirpath, dsn, logs_path, data_volume, verbose, background, artifact
             stderr=open("mlrun-stderr.log", "w"),
             start_new_session=True,
         )
+        pid = child.pid
         print(
-            f"background pid: {child.pid}, logs written to mlrun-stdout.log and mlrun-stderr.log\n"
-            f"use: kill {child.pid}, to stop the mlrun service process (in linux/mac)"
+            f"background pid: {pid}, logs written to mlrun-stdout.log and mlrun-stderr.log, use:\n"
+            f"`kill {pid}` (linux/mac) or `taskkill /pid {pid} /t /f` (windows), to kill the mlrun service process"
         )
     else:
         child = Popen(cmd, env=env)
         returncode = child.wait()
         if returncode != 0:
             raise SystemExit(returncode)
+    if env_file:
+        # update mlrun env file with the API path and PID (for killing it)
+        env_file = mlrun.config.default_env_file if env_file == "$" else env_file
+        filename = path.expanduser(env_file)
+        dotenv.set_key(
+            filename, "MLRUN_DBPATH", f"http://localhost:{port or 8080}", quote_mode=""
+        )
+        if pid:
+            dotenv.set_key(filename, "MLRUN_SERVICE_PID", str(pid), quote_mode="")
+        print(f"updated conviguration in {env_file} .env file")
```

Comment:
IIUC this is being done so users then could use the other CLI command - `mlrun config` to see the what is the pid and the endpoint of the API, if yes i would add rather the scenario you experienced as example as it not that easy to understand why we would even need this.